server/@types/bapv.d.ts

```diff
@@ -19,6 +19,8 @@ export type BookingJourneyData = {
 
   // selected visitors for this visit
   selectedVisitors?: VisitorInfoDto[]
+
+  visitorSupport?: string
```

Comment:
We've got a bit of a mix in the code of `visitorSupport` vs `additionalSupport`: should we standardise on one?

Reply:
we send it as visitor support - that's why I chose this naming